grading/load_config_json.cpp

```diff
@@ -84,6 +84,26 @@ void AddAutogradingConfiguration(nlohmann::json &whole_config) {
   }
 
   if (whole_config["autograding"].find("compilation_to_runner") == whole_config["autograding"].end()) {
+    // copy all executable names from compilation testcases to runner
+    nlohmann::json::iterator testcases = whole_config.find("testcases");
+    for (typename nlohmann::json::iterator testcase = testcases->begin(); testcase != testcases->end(); testcase++) {
+      if (testcase->value("type", "") == "Compilation") {
+        nlohmann::json::iterator exe_name = testcase->find("executable_name");
+        if (exe_name == testcase->end()) {
+          continue;
+        }
+        // Handle single file or array of files.
+        if (exe_name->is_array()) {
+          for (typename nlohmann::json::iterator itr = exe_name->begin(); itr != exe_name->end(); itr++) {
+            whole_config["autograding"]["compilation_to_runner"].push_back(*itr);
+          }
+        }
+        else {
+          whole_config["autograding"]["compilation_to_runner"].push_back(*exe_name);
```

Comment:
Should `**/` go in front of the executable name? I don't think we can be certain where they compiled the file.